README.md

```diff
@@ -6,136 +6,145 @@
 
-```
-var options = new BiometryOptions();
-options.LocalizedReasonBodyText = "REASON THAT APP WANTS TO USE BIOMETRY";
-options.LocalizedFallbackButtonText = "FALLBACK";
-options.LocalizedCancelButtonText = "CANCEL";
-
-// use LAPolicy.DeviceOwnerAuthenticationWithBiometrics for biometrics only with no fallback to passcode/password
-// use LAPolicy.DeviceOwnerAuthentication for biometrics+watch with fallback to passcode/password
-_biometryService = new BiometryService(options, async ct => "Biometrics_Confirm", LAPolicy.DeviceOwnerAuthentication);
+Please note that Encrypt/Decrypt methods are only available when using `.SetAllowedAuthenticators(AndroidX.Biometric.BiometricManager.Authenticators.BiometricStrong)` in the `BiometricPrompt.PromptInfo.Builder` instantiation that is required for the service.
+
+Please also note that the title and subtitle are used for `Fingerprint` and `Face` biometry.
+
+Here is an example of instantiation of the service for Android.
+
+``` cs
+var promptBuilder = () => new BiometricPrompt.PromptInfo.Builder()
+	.SetTitle("Title")
+	.SetSubtitle("Subtitle")
+	.SetNegativeButtonText("Cancel")
+	.SetAllowedAuthenticators(AndroidX.Biometric.BiometricManager.Authenticators.BiometricStrong)
+	.Build();
+
+var biometryService = new BiometryService(
+	fragmentActivity: MainActivity.Instance,
+	promptInfoBuilder: promptBuilder,
+	loggerFactory: null
+);
 ```
 
-#### Android
+#### iOS
 
-Face ID is only available with the followwing configuration `.SetAllowedAuthenticators(BiometricManager.Authenticators.BiometricWeak` during instantiation of the service.
+Please note that you must set `NSFaceIDUsageDescription` (key/value) in the `Info.plist` file otherwise the service may throw exceptions or fail.
 
-Encrypt/Decrypt method are only available with the following configuration `.SetAllowedAuthenticators(BiometricManager.Authenticators.BiometricStrong` during instantiation of the service.
+Please also note that the subtitle is used for `Fingerprint` biometry only.
 
-An example of instantiation is as follow, with the fallback to the pin code with some text descriptions to display for the user.
+Here is an example of instantiation of the service for iOS.
 
-```
+``` cs
 _biometryService = new BiometryService(
-    MainActivity.Instance,
-    CoreDispatcher.Main,
-    ct => Task.FromResult(new BiometricPrompt.PromptInfo.Builder()
-    .SetTitle("Biometrics SignIn")
-    .SetSubtitle("Biometrics Confirm")
-    .SetAllowedAuthenticators(BiometricManager.Authenticators.BiometricStrong | BiometricManager.Authenticators.DeviceCredential) // Fallback on secure pin
-    .SetNegativeButtonText("Cancel")
-    .Build()));
+	useOperationPrompt: "Subtitle",
+	laContext: null,
+	localAuthenticationPolicy: LAPolicy.DeviceOwnerAuthenticationWithBiometrics,
+	loggerFactory: null
+);
 ```
 
 ## Methods
 
+Please note that in case of error, `BiometryException` is thrown. If it's a cancellation error, `OperationCanceledException` is thrown.
```

Comment:
I think it would be worth it explaining all type of biometryexception that can be thrown